src/internal/Subscriber.ts

```diff
@@ -33,9 +35,14 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
                    error?: (e?: any) => void,
                    complete?: () => void): Subscriber<T> {
     const subscriber = new Subscriber(next, error, complete);
+    subscriber.syncErrorThrowable = false;
     return subscriber;
   }
 
+  public syncErrorValue: any = null;
```

Comment:
I'd rather make these as `private` and let other internal just cast and refer it, at least to not to be exposed as TS interfaces for strict indication these are internals. (including below property as well)

Reply:
We can't as there are other parts of the library that need to access it as a property, but I can flag it as `/**@internal*/`.

Reply:
Done.